src/raftstore/store/peer.rs

```diff
@@ -528,7 +528,6 @@ impl Peer {
         }
 
         debug!("{} propose command with uuid {:?}", self.tag, cmd.uuid);
-        PEER_PROPOSAL_COUNTER_VEC.with_label_values(&["all"]).inc();
```

Comment:
We can compute the total number of proposals by `sum(tikv_raftstore_proposal_total)`, `all` is unnecessary.